download.html

```diff
@@ -13,6 +13,16 @@ <h2>Binaries</h2>
 <br>
 </p>
 
+<div class="panel panel-danger">
+  <div class="panel-heading">
+    <h3 class="panel-title">macOS Big Sur is not supported</h3>
+  </div>
+  <div class="panel-body">
+    If you are using macOS, please <em>do not upgrade</em> if ilastik is critical to your workflow.
```

Comment:
upgrade what? ilastik or OSX?

```suggestion
    If you are using macOS, please <em>do not upgrade to macOS 11.0 (Big Sur)</em> if ilastik is critical to your workflow.
```

Reply:
I'll drop `.0` from `11.0` (what if they'll release `11.1` tomorrow?..)